README.md

```diff
@@ -66,141 +66,25 @@ at https://www.azureiotsuite.com.
 Getting Started
 ===============
 
-## Prerequisites
-* [Azure Subscription](https://azure.microsoft.com/free/) (also see [permissions guidelines](https://docs.microsoft.com/azure/iot-suite/iot-suite-permissions))
-
-## Setup
-1. Open a terminal window or command shell
-1. Check out the repository and submodules
-    ```
-    git clone --recursive https://github.com/Azure/azure-iot-pcs-remote-monitoring-java
-    ```
-    If you cloned the repo without the `--recursive` flag and it's your first time updating the submodules, you can use
-    `git submodule update --init --recursive`. More information on submodules [here](https://git-scm.com/book/en/v2/Git-Tools-    Submodules). 
-    
-1. Set up command line interface for deployments
-
-     #### Prerequisite
-    * Install [npm](https://www.npmjs.com/get-npm)
-
-    #### Mac/Linux
-    
-    ```
-    cd azure-iot-pcs-remote-monitoring-java/cli
-    npm install
-    npm start
-    npm link
-    ```
-    #### Windows
-    
-    ```
-    cd azure-iot-pcs-remote-monitoring-java\cli
-    npm install
-    npm start
-    npm link
-    ```
-    
-1. Sign into your Azure account with
-    ```
-    pcs login
-    ```
-
-## Deploy a remote monitoring solution
-### Standard Deployment
-A standard deployment is a production-ready solution for reliability and scale. *See [Basic vs. Standard Deployments](#basic-vs-standard-deployments) for more details.*
-```
-pcs -s standard --runtime java
-```
-
-### Basic Deployment
-A lower-cost showcase solution best for discovering and learning about what a Remote Monitoring preconfigured solution has to offer.  *See [Basic vs. Standard Deployments](#basic-vs-standard-deployments) for more details.*
-```
-pcs --runtime java
-```
-
-### Local Deployment
-To set up a Remote Monitoring Solution locally see
-[Local Development](https://github.com/Azure/azure-iot-pcs-remote-monitoring-dotnet/wiki/Developer-Reference-Guide#local-development)
+You can follow along with the [Getting Started](https://docs.microsoft.com/azure/iot-suite/iot-suite-remote-monitoring-deploy) documentation to learn how to deploy using the web interface. You can also deploy using the [command line](https://docs.microsoft.com/azure/iot-suite/iot-suite-remote-monitoring-deploy-cli).
```

Comment:
formatting nit:
It might be good to break out the two options with bullet points instead of the paragraph format. Typically when I'm in a repo for the first time I either:
1) want to read a lot and read everything, or...
2) get me running as fast as possible 

Perhaps the format could be along the lines of:

## Deploy a solution
There are two ways to deploy a solution:
1) Deploy using the web interface using the instructions [here](https://docs.microsoft.com/azure/iot-suite/iot-suite-remote-monitoring-deploy).
2) Deploy using the [command line](https://docs.microsoft.com/azure/iot-suite/iot-suite-remote-monitoring-deploy-cli).

Reply:
Yes that looks much cleaner to me too